.env.example

```
@@ -60,9 +56,8 @@ TRUST_X_FORWARDED_FOR=false # Only enable behind a trusted reverse proxy
 CORS_ORIGINS=*              # Comma-separated origins; "*" = allow all
 # CORS_SUPPORTS_CREDENTIALS=false  # Cannot combine with wildcard CORS_ORIGINS
 
-# WebSocket authentication (dedicated WS server on port 4001)
-WS_HOST=0.0.0.0             # Bind address for dedicated WS server; 0.0.0.0 accepts all interfaces (same as HTTP server default)
-WS_REQUIRE_TOKEN=false      # Default is false; reachable clients can connect without a token, so keep port 4001 on trusted networks or enable WS_REQUIRE_TOKEN=true
+# WebSocket authentication (/ws on the web server port)
+WS_REQUIRE_TOKEN=false      # Default is false; reachable clients can connect without a token, so keep the web port on trusted networks or enable WS_REQUIRE_TOKEN=true
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)